Core/UserDefaultsPropertyWrapper.swift

```diff
@@ -98,7 +98,7 @@ public struct UserDefaultsWrapper<T> {
 
         case syncEnvironment = "com.duckduckgo.ios.sync-environment"
 
-        case networkProtectionDebugOptionAlwaysOnEnabled = "com.duckduckgo.network-protection.always-on.enabled"
+        case networkProtectionDebugOptionAlwaysOnDisabled = "com.duckduckgo.network-protection.always-on.disabled"
```

Comment:
Changing the user default so that it will be ON by default for existing users.  In any case the change makes sense IMO since the default is now going to be ON, so the presence of this will most likely indicate that the user disabled always on.